### PR TITLE
Very important update to smokebomb description generation

### PR DESCRIPTION
**Pull request by ghost**
## About The Pull Request
	///It's extremely important to keep this list up to date. It helps to generate the insightful description of the smokebomb

## Why It's Good For The Game
	///It's extremely important to keep this list up to date. It helps to generate the insightful description of the smokebomb

## Changelog
:cl:
fix: Smoke bomb description generation is no longer extremely outdated.
/:cl:

### CODE REVIEW

File: code/game/objects/items/grenades/smokebomb.dm [hunk 1]
```diff
@@ -11,7 +11,7 @@
 	inhand_icon_state = "smoke"
 	slot_flags = ITEM_SLOT_BELT
 	///It's extremely important to keep this list up to date. It helps to generate the insightful description of the smokebomb
-	var/static/list/bruh_moment = list("Dank", "Hip", "Lit", "Based", "Robust", "Bruh", "Gamer")
+	var/static/list/bruh_moment = list("Dank", "Hip", "Lit", "Based", "Robust", "Bruh", "Gamer", "Poggers", "Nyan")
```
Comment (Fikou) [inline code/game/objects/items/grenades/smokebomb.dm:14]: nyan is 2012 words how is this updating

Comment (interestingusernam3) [reply code/game/objects/items/grenades/smokebomb.dm:14]: That's how outdated it is man.

**Comment by CyoTheProtogen** [comment]
When the GBP farm

**Comment by Mothblocks** [comment]
im gonna say no only because i think the comment is stupid and we're gonna have pr's like this frequently if our goal is to always keep it fresh with the new flavor of the month

**Pull request closed without merging**